app/src/main/res/layout/fragment_we_are_hiring.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/constraintLayout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/s_60">
+
+
+    <Button
+        android:id="@+id/apply_now"
+        style="@style/App_button"
+        android:layout_width="0dp"
+        android:layout_height="48dp"
+        android:layout_marginStart="@dimen/size_3x"
+        android:layout_marginTop="@dimen/size_4x"
+        android:layout_marginEnd="@dimen/size_3x"
+        android:text="@string/apply_now"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textView2" />
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_marginTop="@dimen/size_4x"
+        app:layout_constraintTop_toBottomOf="@+id/apply_now"
+        app:srcCompat="@drawable/mask_group_20"
+        android:scaleType="fitXY"/>
```

Comment:
Myślę że jak zmienisz na `android:scaleType="fitEnd"` i dodasz `app:layout_constraintBottom_toBottomOf="parent"` będzie to efekt który chcemy

Reply:
![image](https://user-images.githubusercontent.com/80550704/218806743-755edad7-a756-4f2d-b271-a4474f4d41d9.png)

Niestety po dodaniu zaproponowanych komend nie jest lepiej.